plugins/alerts/frontend/public/templates/vue-main.html

```diff
@@ -0,0 +1,295 @@
+                    <template slot-scope="scope">
+                        <div v-html="scope.row.alertName"></div>
+                    </template>
+                </el-table-column> 
+    
+                <el-table-column  min-width="175"  sortable="true"  prop="appNameList" :label="i18n('alert.Application')">
+                </el-table-column>
+    
+                <el-table-column  min-width="415"  sortable="true"  :label="i18n('alert.Condition')">
+                    <template slot-scope="scope" sortable="true">
+                        <span>{{scope.row.condtionText}}</span>
+                    </template>
+                </el-table-column>
+    
+                <el-table-column  min-width="230" sortable="true"  :label="i18n('alert.CreateBy')">
+                    <template slot-scope="scope">
+                        <div class="bu-level">
+                            <div class="bu-level-left">
+                                <div class="is-created-by-col">
+                                    {{scope.row.createdByUser}}
+                                 </div>
+    
+                            </div> 
+                          <div class="bu-level-right">
+                             <cly-more-options size="small" @command="handleHookEditCommand($event, scope)">
+                                <el-dropdown-item v-if="canUpdate" icon="el-icon-document-copy" command="edit-comment">
+                                    {{i18n('alert.Edit')}}
+                                </el-dropdown-item>
+                                <el-dropdown-item v-if="canDelete" icon="el-icon-delete" command="delete-comment">
+                                    {{i18n('alert.Delete')}}
+                                </el-dropdown-item>
+                            </cly-more-options>
+                          </div>
+                        </div>
+                    </template>
+                </el-table-column>
+
+            </template>
+            <template v-slot:bottomline="scope">
+                <cly-diff-helper :diff="scope.diff" @discard="scope.unpatch()" @save="updateStatus(scope)">
+                </cly-diff-helper>
+            </template>
+        </cly-datatable-n>
+    </cly-section>
+    </script>
+    
+    
+    
+<script  type="text/x-template" id="alert-drawer">
+    <cly-drawer
+        @submit="onSubmit"
+        @close="onClose"
+        @copy="onCopy"
+        :title="title"
+        :saveButtonLabel="saveButtonLabel"
+        v-bind="$props.controls">
+        <template v-slot:default="drawerScope">
+            <cly-form-step id="alert-drawer-main">
+            
+            <div class="bu-py-4">
+                <div class="text-medium text-heading title">
+                    {{i18n('alert.Alert_Name')}}
+                </div>
+                <validation-provider name="alertName" rules="required" v-slot="v">
```

Comment:
instead of directly using the validation-provider, you could use **cly-form-field**
its ok for now but for future work, you should use **cly-form-field**